lib/ssh_scan/api.rb

```diff
@@ -99,25 +123,15 @@ def add(*args, &block)
       }.to_json
     end
 
-
     namespace "/api/v#{SSHScan::API_VERSION}" do
       before do
         content_type :json
       end
 
       post '/scan' do
-        options = {
-          :sockets => [],
-          :policy => File.expand_path("../../../policies/mozilla_modern.yml", __FILE__),
-          :timeout => 2,
-          :verbosity => nil,
-          :logger => NullLogger.new,
-          :fingerprint_database => "fingerprints.db",
-        }
-        options[:sockets] << "#{params[:target]}:#{params[:port] ? params[:port] : "22"}"
-        options[:policy_file] = SSHScan::Policy.from_file(options[:policy])
-        scan_engine = SSHScan::ScanEngine.new()
-        scan_engine.scan(options).to_json
+        # TODO: there needs to be some sort of ID tracking here so the user can come back later and retrieve the results of their scan
```

Comment:
I think we'd need some sort of database if we are going to let users retrieve the results  of their previous scans.